lib/elixir/lib/kernel.ex

```diff
@@ -932,6 +937,11 @@ defmodule Kernel do
   Removes the first occurrence of an item on the left list
   for each item on the right.
 
+  The complexity of `a -- b` is proportional to `length(a) * length(b)`,
```

Comment:
We should note that this is the worst-case complexity. The ideal case is linear.